entries/o/onelogin.com.json

```diff
@@ -1,15 +1,23 @@
 {
   "OneLogin": {
     "domain": "onelogin.com",
-    "url": "https://www.onelogin.com/",
     "tfa": [
       "custom-software",
       "sms",
       "totp",
       "custom-hardware",
       "u2f"
     ],
-    "documentation": "https://onelogin.service-now.com/support?id=kb_article&sys_id=43d99143db109700d5505eea4b96197a",
+    "custom-software": [
+      "OneLogin Protect",
+      "Duo",
+      "RSA SecurID",
```

Comment:
I can't see any mention of SecurID on their help page

Reply:
In step 3 of enabling authentication factors, there's a list below the screenshot. If you click on 'RADIUS' under partners, it's listed there. RADIUS adds support for it